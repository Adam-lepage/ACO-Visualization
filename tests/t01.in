input1.txt
0
20
10
5
0.5