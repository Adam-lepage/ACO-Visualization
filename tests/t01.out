The best found solution is of length 5, which used the following path:
1. 0
2. 2
3. 4
4. 1
5. 3